llm/falcon_h1/falcon_h1.yaml

```
@@ -0,0 +1,51 @@
+# Falcon H1 Series Deployment with vLLM on SkyPilot
+# Supports: Falcon-H1-0.5B, 1.5B, 3B, 34B models
+
+resources:
+  accelerators: [A100:8]  # Use [A100-80GB:8] for 34B model
```

For single GPU, may be easier to use non-list syntax
```suggestion
  accelerators: A100:8  # Use A100-80GB:8 for 34B model
```